sql-files/main.sql

```diff
@@ -965,6 +966,7 @@ CREATE TABLE IF NOT EXISTS `rodex_mail` (
 	`zeny` BIGINT(20) NOT NULL,
 	`type` TINYINT(8) UNSIGNED NOT NULL,
 	`is_read` TINYINT(8) NOT NULL,
+  `sender_read` TINYINT(2) NOT NULL,
```

Comment:
look like formatting bit wrong. probably here should be tab before `sender_read` TINYINT(2) NOT NULL,